### ISSUE TITLE
OrphanedThreadException when connected to CC: Tweaked Remote PC

### ISSUE THREAD

**Issue by n-krueger** [MCJack123/vscode-craftos-pc#16]
First of all great work on this extension! Unfortunately I am encountering the following issue after a short duration of being connected to a CC: Tweaked PC on a multiplayer server:

1. Connect to CC: Tweaked PC with the "Connect to Remote (Beta)" feature
2. Editing files and using the terminal from VSCode works
3. Wait a short while (~1min)
4. VSCode shows notification "Disconnected from WebSocket server", files no longer available, terminal in VSCode no longer responds to key input
5. CC: Tweaked PC in Minecraft shows the following exception:
```
[string "server.lua"]:1: Java Exception Thrown:
org.squiddev.cobalt.InterruptedError:
org.squiddev.cobalt.OrphanedThreadException:
OrphanedThread
```
At this point it is possible to reconnect again, but the exception occurs every time after a short duration.

## Version Details
```
VSCode-CraftOS-PC: v1.2.1
VSCode:            1.77.0 (On MacOS Ventura 13.3, x86)
Modpack:           FTB Revelation 3.6.0
Minecraft Forge:   14.23.5.2858
CC: Tweaked:       1.89.2
```
The Server is hosted on exaroton.com and is using Java 8.

**Comment by MCJack123** [comment]
Unfortunately, this is an issue with CC:T on 1.12 that I can't fix (cc-tweaked/CC-Tweaked#460). It's only fixed on 1.15+, so I'm sorry to say you're SOL.

**Comment by n-krueger** [comment]
Alright, thanks for the pointer to the CC: Tweaked issue!
